README.md

```diff
@@ -63,42 +60,59 @@ _How to get data from other services into RemNote._
 - Airr: [How to setup the Airr + RemNote integration](https://www.youtube.com/watch?v=Y92mPSugGBs) - Import highlights from podcast.
 - Zotero: [remnote-zotero-import](https://github.com/hannesfrank/remnote-zotero-import) - Paste literature references.
 
-### Development
+## Extensions
+### Plugins
 
-_Things a plugin developer might need._
+_Plugins extend the functionality of RemNote, featured plugins (⭐) can be installed directly from [remnote.io/plugins](https://www.remnote.io/plugins)._
 
-- [remnote-inspect](https://github.com/hannesfrank/remnote-inspect) - Show datastructure of context, focused Rem and Document.
+<!-- TODO: describe how to install plugins that aren't featured in the app -->
 
-<!-- TODO: Guides, Github pages plugin template,  -->
 
-## Custom CSS
+- ⭐ [Image Search](https://remnoteimagesearch.glitch.me/) - Search and insert images powered by Google Image Search.
+- ⭐ [RemNote Graph](https://www.nhanvu327.com/rem-plugins) - Graph Visualization for you knowledge base.
+  <small>Implementation currently not published on its [Github](https://github.com/nhanvu327/rem-graph).</small>
+- ⭐ Word Cloud - Visualize a document as a word cloud.
+- ⭐ Word Counter - Count the number of words on the page.
+- [RemNote-Pomodoro](https://github.com/sirvan3tr/RemNote-Pomodoro) - Pomodoro timer and activity logger.
+- [remnote-schedule](https://github.com/hannesfrank/remnote-schedule) - Visualize your daily schedule. Inspired by [CrushEntropy.com](https://crushentropy.com/).
+- [English Dictionary](img/dictionary-plugin.png) - Look up definition of words
+  - Note: As RemNote Plugins are just webpages, you can use any webservice which works well in an iframe.
+
+<!-- TODO: Can we get a link (remnote.io/plugins/add?name=myplugin&width=) to install a plugin with default settings. This would be a first and simple to implement step to a plugin store.
+Martin said one can ask the devs to add a plugin.
+-->
 
-_RemNotes appearance can be customized on the `⮉ Custom CSS` page._
 
 ### Themes
-
-- **[RemNoteTheme](https://github.com/ethomasv/RemNoteTheme)** - De facto standard community theme supporting light and dark mode.
+Installed using [Stylus.](https://github.com/openstyles/stylus)
```

Comment:
Stylus has autoupdating and configuration options built in. AFAIK remnote css blocks are on or off only.

Reply:
CSS variables are like this:
```
:root {
  --color-highlight-red: darkred;
}
```
When installing something you would make 2 blocks. One with an also auto updating `@import` link and one with a config block. IDK if this is would be too hard for a non-techy user.
And the if you split the theme into more blocks it creates a bit of modularity. Eva does that in her youtube video.

Reply:
Would RemNote run those updates from the Custom CSS page?.. Plus, people are used to Stylus and Violentmonkey UIs and one-click installs.

Reply:
>Would RemNote run those updates from the Custom CSS page?
They are imported again at least when you reload RemNote. We could ask Martin if he would implement a periodic refreshment. It's not too difficult to implement.

>  Plus, people are used to Stylus and Violentmonkey UIs and one-click installs.
Are they really? IDK, I'll take your word for it :D

(I had planned for remnote-library to have one-click install as well. The vision is to have a plugin/website where people select what they want (a plugin could also preview stuff directly on some demo rems) and install it directly to their Custom CSS. This is blocked currently blocked because I can not create code blocks via API and code blocks can't be indented.)